content/800-data-platform/050-data-proxy/700-prisma-cli-with-data-proxy.mdx

```diff
@@ -9,19 +9,17 @@ tocDepth: 3
 
 Currently, you can only use [Prisma Client](/concepts/components/prisma-client) with the Data Proxy to _query_ your database.
 
-</TopBlock>
-
+You need a direct (non-Data Proxy) database connection if you apply schema changes to your database with [Prisma Migrate](/concepts/components/prisma-migrate/get-started) or [`db push`](/concepts/components/prisma-migrate/db-push), or if you [introspect](/concepts/components/introspection) your database. See [Prisma CLI commands that require a direct database connection](#prisma-cli-commands-that-require-a-direct-database-connection).
```

Comment:
Upon rereading this, we could make clearer that Prisma Client is totally fine. 
And that it is only Prisma CLI where commands are not by default and the additional feature here are needed.
Right now that is not super spelled out, both in words and visually.

(Line 12 could maybe start with "In Prisma CLI you need...")